internal/testutils/jimmtest/mocks/jimm_offerauthorizer_mock.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2025 Canonical.
+
+package mocks
+
+import (
+	"context"
+
+	"github.com/juju/names/v5"
+)
+
+// OfferAuthorizer is a mock implementation of the OfferAuthorizer interface.
+type OfferAuthorizer struct {
```

Comment:
Although we're not using it, we should probably also add a getter for the offer authorizer in `internal/testutils/jimmtest/jimm_mock.go`.